### DISCUSSION TITLE
Github Academic Status Page Keeps Refreshing

### DISCUSSION THREAD

**Discussion by GlendaChong** [community/community#65909] [Copilot Conversations]
### Select Topic Area

Bug

### Body

Hi, I am trying to get approval for my academic status to the Github Student Developr Pack. I've uploaded the image, and clicked process application. The application shows to be processed, where it finished at 100%, but then the entire page refreshes, and I do not receive any confirmation that my application has been processed or approved. 

May I know if there is a fix on this, and is my application submitted?

**Comment by arafajs** [comment]
Try a different browser.
